site/content/installation/nginx-plus-jwt.md

```diff
@@ -0,0 +1,240 @@
+
+Specify the endpoint in the `--usage-report-endpoint` command-line flag on the `nginx-gateway` container. You also need to add the following line to the `mgmt` block of the `nginx-includes-bootstrap` ConfigMap:
+
+```text
+usage_report endpoint=<your-endpoint>;
+```
+
+{{% /tab %}}
+
+{{</tabs>}}
+
+#### CA and Client certificate/key {#nim-cert}
+
+To configure a CA cert and/or client certificate and key, a few extra steps are needed.
+
+First, you need to create two Secrets in the `nginx-gateway` namespace. The CA must live under the key `ca.crt`:
+
+```shell
+kubectl -n nginx-gateway create secret generic nim-ca --from-file ca.crt
+```
+
+The client cert and key must be added to a TLS Secret:
+
+```shell
+kubectl -n nginx-gateway create secret tls nim-client --cert /path/to/cert --key /path/to/key
+```
+
+{{<tabs name="nim-secret-install">}}
+
+{{%tab name="Helm"%}}
+
+Specify the CA Secret name using the `nginx.usage.caSecretName` helm value. Specify the client Secret name using the `nginx.usage.clientSSLSecretName` helm value.
+
+{{% /tab %}}
+
+{{%tab name="Manifests"%}}
+
+Specify the CA Secret name in the `--usage-report-ca-secret` command-line flag on the `nginx-gateway` container. Specify the client Secret name in the `--usage-report-client-ssl-secret` command-line flag on the `nginx-gateway` container.
+
+You also need to define the proper volume mounts to mount the Secrets to the nginx container. Add the following volumes to the Deployment:
+
+```yaml
+- name: usage-client-ssl-secret
+  secret:
+    secretName: nim-client
+- name: usage-ca-secret
+  secret:
+    secretName: nim-ca
+```
+
+and the following volume mounts to the `nginx` container:
+
+```yaml
+- mountPath: /etc/nginx/usage-certs/client
+  name: usage-client-ssl-secret
+- mountPath: /etc/nginx/usage-certs/ca
+  name: usage-ca-secret
+```
+
+Finally, in the `nginx-includes-bootstrap` ConfigMap, add the following lines to the `mgmt` block:
+
+```text
+ssl_trusted_certificate /etc/nginx/usage-certs/ca/ca.crt;
+ssl_certificate        /etc/nginx/usage-certs/client/tls.crt;
+ssl_certificate_key    /etc/nginx/usage-certs/client/tls.key;
+```
+
+{{% /tab %}}
+
+{{</tabs>}}
+
+<br>
+
+**Once these Secrets are created and configuration options are set, you can now [install NGINX Gateway Fabric]({{< relref "installation/installing-ngf" >}}).**
+
+## Installation flags for configuring usage reporting {#flags}
+
+When installing NGINX Gateway Fabric, the following flags can be specified to configure usage reporting to fit your needs:
+
+If using Helm, the `nginx.usage` values should be set as necessary:
+
+- `secretName` should be the name of the JWT Secret you created. By default this field is set to `nplus-license`. This field is required.
+- `endpoint` is the endpoint to send the telemetry data to. This is optional, and by default is `product.connect.nginx.com`.
+- `resolver` is the resolver domain name or IP address with optional port for resolving the endpoint.
```

Comment:
I'm assuming this field is optional? Does it only apply to NIM?

Reply:
Correct

Reply:
Probably worth mentioning in the doc string